lib/core/http.js

```
@@ -74,5 +74,31 @@ module.exports = (config) => {
     );
   };
 
-  return { getCageKey, runCage, getCert };
+  const postMetric = async (count) => {
+    const metricsUrl = `metrics/sdkEncryptions?sdk=node&numEncryptions=${count}`;
+    try {
+      await post(metricsUrl);
+    } catch (e) {}
```

again not good to swallow errors